chrome/content/zotero/components/createParent/createParent.jsx

```diff
@@ -25,12 +25,17 @@
 
 'use strict';
 
-import React, { memo } from 'react';
+import React, { memo, useEffect } from 'react';
 import ReactDOM from "react-dom";
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
 function CreateParent({ loading, item, toggleAccept }) {
+	// With React 18, this is required for the window's dialog to be properly sized
+	useEffect(() => {
+		window.sizeToContent();
+	});
```

Comment:
Even though it doesn't matter for this component, as I don't think it can re-render (no state, props don't change), I'd recommend adding empty array as the `dependencies` argument to the `useEffect` to ensure that logic of the effect is executed once when component mounts and not on every update.